flytekit/image_spec/default_builder.py

```diff
@@ -247,6 +257,7 @@ class DefaultImageBuilder(ImageSpecBuilder):
         "pip_extra_index_url",
         # "registry_config",
         "commands",
+        "copy_src_dest",
```

Comment:
Could we change the name to `copy`? Just make it consistent with the command in the dockerfile.

Reply:
Since we now have a function called `ImageSpec.copy()`, naming the argument `copy` might be confusing. Maybe rename the copy function to `copy_to_root` or name the argument something like `copy_src` will be a solution.

Reply:
I will change the function name to `ImageSpec.with_copy()` and the argument name to `copy`. wdyt @thomasjpfan